src/components/PageHeader/PageHeader.scss

```diff
@@ -0,0 +1,50 @@
+@import '~@quartz/styles/scss/borders';
+@import '~@quartz/styles/scss/color-scheme';
+@import '~@quartz/styles/scss/fonts';
+@import '~@quartz/styles/scss/media-queries';
+
+.container {
+	text-align: center;
+
+	&.border {
+		border-bottom: $border-solid-decorative;
+	}
+}
+
+.contents {
+	margin-top: 20px;
+	margin-bottom: 25px;
+
+	@include for-tablet-portrait-up {
+		margin-top: 30px;
+		margin-bottom: 50px;
+	}
+
+	@include for-tablet-landscape-up {
+		margin-top: 50px;
+		margin-bottom: 60px;
+	}
```

Comment:
I suspect these media queries and values are pretty arbitrary, could you check with Daniel whether we can simplify?

Reply:
margin
```suggestion
	@include for-tablet-portrait-up {
		margin-top: 48px;
		margin-bottom: 60px;
	}

	@include for-tablet-landscape-up {
		margin-top: 48px;
		margin-bottom: 60px;
	}
```

Reply:
Thank you!